### ISSUE TITLE
Grey out the extension icon when pronoun replacement is being prevented

### ISSUE THREAD

**Issue by beszel** [glam-lab/degender-the-web#96]
**Is your feature request related to a problem? Please describe.**
Currently, if the extension won't replace pronouns on a page, the header says so. If the DGtW header is going to be disabled or removed entirely (see #68), there should be another indication that the extension is prevented from replacing pronouns. The icon should also be greyed out on system pages such as the extensions page, as the extension can't run on those.

**Describe the solution you'd like**
The extension icon should be greyed out when the extension is prevented from replacing pronouns on a page. The icon should remain colored on pages where gendered pronouns have been replaced, as well as on pages where gendered pronouns *would* be replaced if the page had any. Other extensions do something similar, most obviously on system pages. This solution conveys an important bit of information without introducing visual clutter.

**Describe alternatives you've considered**
The extension popup will explain why the page isn't replacing pronouns, but it would still be nice to check at a glance whether the extension is working for the current page.
The icon could also be greyed out when there are no gendered pronouns on the page, but I think that would mislead the user into thinking that the extension is prevented from replacing pronouns. The user might think that the extension has incorrectly identified the page as that discusses gender.

**Comment by ProfJanetDavis** [comment]
I like your reasoning here! This seems like a good follow-up to #68.

**Comment by beszel** [comment]
I created a set of three grey icons that can be used for now. The icon's color is associated with each tab, so switching from a Wikipedia tab to a Facebook tab also switches the color icon for the grey icon. I had to add a background page to help the content script control the icon.

Wikipedia, whether or not replacements have been made (see OP):
![icons-wikipedia](https://user-images.githubusercontent.com/23243513/68504810-b78b8200-021a-11ea-951e-e9d37aa3caf2.png)
Facebook (does not run) or Wikipedia after restoring original content:
![icons-facebook](https://user-images.githubusercontent.com/23243513/68504783-a2aeee80-021a-11ea-9725-f251cc3b29ba.png)
Here's how it fits in with other disabled icons on system pages.
![icons-system-page](https://user-images.githubusercontent.com/23243513/68504784-a3478500-021a-11ea-8308-9030870de1d9.png)

Note that I'm using Wikipedia as an example of a normal site. Other sites will show the same behavior.

This didn't take much work, so any future changes to icon behavior should be similarly quick.

From looking at other extensions, it seems I'll need a grey icon for each of the three icon sizes. I made the smallest icon grey by default to handle system pages, but I'll need to do something about the larger icons. I can't just set the larger icons to be grey by default because one of them is used on the extension page, and that one should appear full color.
